jenkins/ompi/ompi_test.sh

```diff
@@ -110,265 +103,226 @@ mlnx_cov="SKIP"
+
+                if [ "${ci_test_xrc}" = "yes" ]
+                then
+                    # shellcheck disable=SC2086
+                    ${timeout_exe} "$mpirun" --np "$np" $mca --mca pml ob1 --mca btl self,openib --mca btl_openib_receive_queues X,4096,1024:X,12288,512:X,65536,512 \
+                        ${AFFINITY} "${exe_path}" "${exe_args}"
                 fi
             fi
 
-#if [ "$hca_dev" = "mlx4_0" ]; then
-#                rdma_opt="-mca btl_openib_receive_queues P,65536,256,192,128:S,128,256,192,128:S,2048,1024,1008,64:S,12288,1024,1008,64:S,65536,1024,1008,64"
-#                $timeout_exe $mpirun -np $np $common_mca $rdma_opt -mca btl_openib_cpc_include rdmacm -mca pml ob1 -mca btl self,openib -mca btl_openib_if_include ${hca_dev}:2 \
-#                ${AFFINITY} ${exe_path} ${exe_args}
-#            fi
-
-            if [ "$jenkins_test_ucx" = "yes" -a $has_ucx -gt 0 -a "$hca_dev" != "mlx4_0" ]; then
-                $timeout_exe $mpirun --np $np $mca --mca btl self --mca pml ucx ${AFFINITY} ${exe_path} ${exe_args}
-            fi
-            if [ $has_yalla -gt 0 ]; then
-                $timeout_exe $mpirun --np $np $mca --mca btl self --mca pml yalla ${AFFINITY} ${exe_path} ${exe_args}
-            fi
-            if [ -n "$mpi_custom_args" ]; then
-                $timeout_exe $mpirun --np $np $mca --mca btl self $mpi_custome_args ${AFFINITY} ${exe_path} ${exe_args}
+            if [ "${ci_test_ucx}" = "yes" ] && [ "${has_ucx}" -gt 0 ] && [ "${hca_dev}" != "mlx4_0" ]
+            then
+                # shellcheck disable=SC2086
+                ${timeout_exe} "$mpirun" --np "$np" $mca --mca btl self --mca pml ucx ${AFFINITY} "${exe_path}" "${exe_args}"
             fi
         fi
     done
 }
 
 function oshmem_runner()
 {
-
     AFFINITY=${AFFINITY_GLOB}
-    if [ "$1" = "--no-bind" ]; then
+
+    if [ "$1" = "--no-bind" ]
+    then
         AFFINITY=""
         shift
     fi
 
     local np=$1
     local exe_path="$2"
     local exe_args=${3}
-    local spml_yoda="--mca spml yoda"
-    local spml_ikrit="--mca spml ikrit"
     local spml_ucx="--mca spml ucx"
-    local oshrun="$OMPI_HOME/bin/oshrun"
-    local common_mca="--bind-to none -x SHMEM_SYMMETRIC_HEAP_SIZE=256M --mca orte_tmpdir_base $jenkins_session_base"
+    local oshrun="${OMPI_HOME}/bin/oshrun"
+    local common_mca="--bind-to none -x SHMEM_SYMMETRIC_HEAP_SIZE=256M --mca orte_tmpdir_base ${ci_session_base}"
 
-    local has_ucx=$($OMPI_HOME/bin/ompi_info --param pml all --level 9 | grep ucx | wc -l)
+    local has_ucx
+    has_ucx=$("${OMPI_HOME}/bin/ompi_info" --param pml all --level 9 | grep --count ucx || true)
 
-    local has_timeout=$($OMPI_HOME/bin/mpirun --help | grep timeout | wc -l)
-    if [ "$has_timeout" = "0" ]; then
+    local has_timeout
+    has_timeout=$("${OMPI_HOME}/bin/mpirun" --help | grep --count timeout || true)
```

Comment:
no need to check for internal timeout, it always exist now

Reply:
Fixed.